source/hwIo.py

```diff
@@ -175,8 +175,7 @@ def __init__(self, *args, **kwargs):
 		self._origTimeout = self._ser.timeout
 		# We don't want a timeout while we're waiting for data.
 		self._setTimeout(None)
-		self.inWaiting = self._ser.inWaiting
-		super(Serial, self).__init__(self._ser.hComPort, onReceive)
+		super(Serial, self).__init__(self._ser._port_handle, onReceive)

```

Comment:
This handle is now private to pyserial, but if there is no public function to retrieve it I guess this is the best we can do.

Reply:
I did an extra check, but there doesn't seem to be one.